src/components/TodoItem.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+
+
+export default class TodoItem extends Component {
+
+    render() {
+        const {id, title } = this.props.todo
+        return (
+            <div style={itemStyle} >
+                <p>
+                <input type="checkbox" onChange={this.props.markComplete.bind(this, id)} /> {' '}
+                { title }
+                <button onClick={this.props.delTodo.bind(this, id)} style={btnStyle}>x</button>
```

Comment:
no need of using bind use arrow function to declare delTodo()
change it has:-
onClick={()=>this.props.delTodo(id)}